studio/components/interfaces/Auth/AutoSchemaForm.tsx

```diff
@@ -65,22 +70,27 @@ const AutoSchemaForm = observer(() => {
     payload.DISABLE_SIGNUP = !values.DISABLE_SIGNUP
 
     setSubmitting(true)
-    const { error } = await authConfig.update(payload)
-
-    if (!error) {
-      ui.setNotification({
-        category: 'success',
-        message: `Successfully updated settings`,
-      })
-      resetForm({ values: values, initialValues: values })
-    } else {
-      ui.setNotification({
-        category: 'error',
-        message: `Failed to update settings:  ${error?.message}`,
-      })
-    }
-
-    setSubmitting(false)
+    updateAuthConfig(
+      { projectRef: projectRef!, config: payload },
+      {
+        onError: (error) => {
```

Comment:
likewise as per https://github.com/supabase/supabase/pull/16227/files#r1289813556
shift all these to the mutation declaration

Reply:
Can't, due to `resetForm`.